tests/data/placebo/test_glue_catalog_cross_account/glue.GetResourcePolicy_1.json

```diff
@@ -0,0 +1,28 @@
+{
+    "status_code": 200,
+    "data": {
+        "PolicyInJson": "{\n  \"Version\" : \"2012-10-17\",\n  \"Statement\" : [ {\n    \"Effect\" : \"Allow\",\n    \"Principal\" : {\n      \"AWS\" : \"arn:aws:iam::123123456789:root\"\n    },\n    \"Action\" : \"glue:CreateTable\",\n    \"Resource\" : \"arn:aws:glue:us-east-1:644160558196:catalog\"\n  } ]\n}",
```

Comment:
The account number in the Principal would get overridden to our default account here. I changed it to `123123456789` which originally was something else(external AWS account) to support the test case. Let me know if that makes sense to you!! Thanks

Reply:
yeah.. we need a flag to disable the account rewrite stuff